src/database/services/PanelService.ts

```diff
@@ -55,6 +61,23 @@ class PanelService {
             panel_set_id: panel.panel_set_id
         } as PanelInfoGet
     }
+
+    /**
+     * Get all panels that are associated with a specific panelSet
+     * @param {number} panel_set_id ID of panelSet
+     * @returns {object[]} An array of objects with id, image, index properties
+     */
+    static async getPanelsFromPanelSetID(panel_set_id: number){
+        // Find all panels on requested panelSet 
+        const panels = await Panel?.findAll({where: {panel_set_id: panel_set_id}});
```

Comment:
```suggestion
        const panels = await Panel.findAll({where: {panel_set_id}});
```
Can also be this. I don't think the `?.` is necessary, safe to assume Panel exists. Also there's some easy sugar there.